src/components/container/MealEditor/MealEditor.stories.tsx

```
@@ -1,63 +1,102 @@
-        </Card>
-    </Layout>;
+        return <Layout colorScheme={args.colorScheme}>
+            <Card>
+                <MealEditor
+                    previewState={computePreviewState(args)}
+                    onError={() => console.log('error')}
+                    onDelete={() => console.log('delete')}
+                    onCancel={() => console.log('cancel')}
+                    onSave={() => console.log('save')}
+                    withImageCapture={args.withImageCapture}
+                    children={args.withChildren ? createChildren() : undefined}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)